### PR TITLE
Migrated to unified build script

### PR DESCRIPTION
**Pull request by SinTh0r4s**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,72 +1,521 @@
...
+/*
+DO NOT CHANGE THIS FILE!
+
+Also, you may replace this file at any time if there is an update available.
+Please check https://github.com/SinTh0r4s/ExampleMod1.7.10/blob/main/build.gradle for updates.
+ */
+
+
+import com.github.jengelman.gradle.plugins.shadow.tasks.ConfigureShadowRelocation
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
```
Comment (Glease) [inline build.gradle:11]: There is nothing to shadow. I prefer to not have this plugin here.

Comment (SinTh0r4s) [reply build.gradle:11]: The plugin is provided by the unified build script. It will always be present so we can have one build script for all mods

**Review by Glease — CHANGES_REQUESTED** [review]
There is already a github action to do automated builds. Please don't keep both in.

**Comment by SinTh0r4s** [comment]
Good catch! Thanks

**Review by Glease — APPROVED** [review]

**Pull request merged**